src/Libraries/CoreNodeModels/Input/BoolSelector.cs

```diff
@@ -37,7 +37,8 @@ public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode
     [NodeName("Boolean")]
     [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
     [NodeDescription("BooleanDescription", typeof(Properties.Resources))]
-    [NodeSearchTags("BooleanSelectorSearchTags", typeof(Properties.Resources))]
+    [NodeSearchTags("BooleanSelectorSearchTags", 
+typeof(Properties.Resources))]
```

Comment:
Why is this file changed? there are no code changes, do we really need `using DSCoreNodesUI.Properties`?